src/.env.template

```diff
@@ -1,2 +1,3 @@
 NUXT_PUBLIC_GTAG_ID=
 NUXT_PUBLIC_TURNSTILE_SITE_KEY=
+NUXT_PRIVATE_OPENAI_API_KEY=
```

Comment:
This new environment variable must be made available to the deployment / stack as well, see https://github.com/maevsi/maevsi_stack/blob/6e058dfc15e72b0ecb6528e40693afe977fe944c/src/development/stack.yml#L190 as an example (ignore the TODO comment about secret files there, it's not really up to date anymore).

Right now, in pure Nuxt development with no stack started, it was sufficient to just create a `.env` file in the Nuxt project and place the api key into it. For the production deployment that change has to be made in maevsi_stack as well, in a similar file close to where the link above is pointing. Take this as a challenge to find out how and where to place this variable in maevsi_stack or ask me for an explanation if you're up for a different challenge.